File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task17_PERPLEXITY_llama-3-sonar-large-32k-chat.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>

char* toJadenCase(char* phrase) {
    if (phrase == NULL || phrase[0] == '\0') return NULL;
    char* p = phrase;
    while (*p != '\0') {
        if (p == phrase || *(p-1) == ' ') {
            *p = toupper((unsigned char)*p);
        } else {
            *p = tolower((unsigned char)*p);
        }
        p++;
    }
    return phrase;
}

int main() {
    char phrase[] = "How can mirrors be real if our eyes aren't real";
    char* jadenCase = toJadenCase(phrase);
    printf("%s
", jadenCase);
    return 0;
}

File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task17_PERPLEXITY_llama-3-sonar-large-32k-chat.java
public class JadenCase {
    public static String toJadenCase(String phrase) {
        if (phrase == null || phrase.isEmpty()) return null;
        String[] words = phrase.split(" ");
        for (int i = 0; i < words.length; i++) {
            words[i] = words[i].substring(0, 1).toUpperCase() + words[i].substring(1).toLowerCase();
        }
        return String.join(" ", words);
    }
    public static void main(String[] args) {
        System.out.println(toJadenCase("How can mirrors be real if our eyes aren't real"));
    }
}